docs: Update index.en-US.md (#52614)
ant-design/ant-design: components/input/index.en-US.md

| Property | Description | Type | Default |
| --- | --- | --- | --- |

> | enterButton | Passing `false` will display the button in its default color, while passing `true` will display the button in its primary color. Alternatively, a custom button can be provided. This property conflicts with the `addonAfter` property | ReactNode | false |

Reviewer: ```suggestion
| enterButton | false displays the default button color, true uses the primary color, or you can provide a custom button. Conflicts with addonAfter. | ReactNode | false |
```
